<?xml version="1.0"?>
<launch>
  <arg name="point_cloud" default="cam3d/depth_registered/points"/>
  <arg name="triggered_point_cloud" default="action_trigger/point_cloud"/>
  <arg name="bounding_box_array" default="tabletop_object_cluster/bounding_box_array"/>

  <node pkg="cob_table_object_cluster" type="table_object_cluster_action_server_node" name="table_object_cluster_action_server" output="screen">
    <remap from="point_cloud" to="$(arg point_cloud)"/>
    <remap from="triggered_point_cloud" to="$(arg triggered_point_cloud)"/>
    <remap from="bounding_box_array" to="$(arg bounding_box_array)"/>
    <remap from="start_pc_sub" to="tabletop_object_cluster/start_pc_sub"/>
  </node>
</launch>
